.spelling

```diff
@@ -479,6 +479,54 @@ embeddable
 p99
 vCPU
 AES-NI
+3.99ms
+72.96ms
+18x
+8x
+appswitch
+appswitch-0
+AppSwitch
+bitpipe
+DaemonSet
+datapath
+decap
+decapsulated
+delayerification
```

Comment:
layerification is not a proper word either. I would remove all uses of this and use a proper expansion

Reply:
It's not supposed to be a proper word but I am not happy with its sound either.  I'll remove it.